client/src/main/java/jass/client/controller/LobbyController.java

```diff
@@ -1,32 +1,63 @@
+/*
+ * fhnw-jass is jass game programmed in java for a school project.
+ * Copyright (C) 2020 Manuele Vaccari
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
 package jass.client.controller;
 
 import com.jfoenix.controls.JFXButton;
+
+
+import jass.client.entity.LoginEntity;
+import jass.client.eventlistener.GameFoundEventListener;
+import jass.client.message.CancelSearchGame;
+import jass.client.message.SearchGame;
 import jass.client.mvc.Controller;
-import jass.client.util.I18nUtil;
 import jass.client.util.SocketUtil;
+import jass.lib.message.CancelSearchGameData;
+import jass.lib.message.SearchGameData;
+import jass.lib.servicelocator.ServiceLocator;
+import javafx.event.ActionEvent;
+import javafx.scene.control.Alert;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import jass.client.util.I18nUtil;
 import jass.client.util.ViewUtil;
 import jass.client.util.WindowUtil;
 import jass.client.view.LobbyView;
-import jass.lib.servicelocator.ServiceLocator;
 import javafx.application.Platform;
 import javafx.fxml.FXML;
 import javafx.scene.control.Menu;
 import javafx.scene.control.MenuItem;
 import javafx.scene.input.KeyCombination;
 import javafx.scene.text.Text;
 
-
 import java.net.URL;
 import java.util.ResourceBundle;
 
 /**
  * The controller for the lobby view.
  *
- * @author Sasa Trajkova
+ * @author Manuele Vaccari & Thomas Weber & Sasa Trajkova
```

Comment:
maybe it would be better to just comment the methods that we authored instead of a whole class